core/src/main/java/org/apache/accumulo/core/clientImpl/bulk/Bulk.java

```diff
@@ -64,8 +67,11 @@ public Files getFiles() {
    */
   public static class Tablet {
 
-    private final byte[] endRow;
-    private final byte[] prevEndRow;
+    private byte[] endRow;
```

Comment:
I am not sure but if the final keyword causes issues without using Unsafe, although it would still maybe cause weird behavior so still not a good idea since using reflection.

However, the thing that does cause problems is not having a default constructor. Adding in a default constructor means we can't use final so it has to be removed either way.

Reply:
> Adding in a default constructor means we can't use final so it has to be removed either way.

Nice, I didn't think about the default constructor so that would cause a compile time error if someone added final.